CONTRIBUTING.md

```diff
@@ -1,5 +1,16 @@
 # Contributing
 
+## Tips on Submiting Pull Requests
+
+1. When you write your Commit messages, please adhere to [CommitConventions](https://www.conventionalcommits.org/en/v1.0.0/)
+2. Make sure your code conforms to all of the auto stylers that we use:
+   -- .prettierrc
+   -- .eslintrc.js
+   -- .editorconfig
+   If you would like to request a code style change, you must justify your reason, provide the relevant rule for the config file and, get team approvel.
```

Comment:
We can add him to use this cmd `npm run format`
It's format automaticaly to the code style.